src/main/java/capstone/mfslbackend/response/dto/TeamResponse.java

```diff
@@ -23,4 +23,17 @@ public class TeamResponse implements Serializable {
     String country;
     Integer founded;
     Boolean national;
+
+    public TeamResponse(long id, String name, String logo, String update, Boolean winner, String code, String country, Integer founded, Boolean national) {
```

Comment:
Dont know why you replaced all the tags with actual constructors

Reply:
For context, its better to just use the tag because if we add or remove any attributes from the class, we dont have to go and change the constructor or the getters and setters